pkg/server/routes.go

```diff
@@ -157,16 +157,18 @@ func (s *Server) routes(
 		ldClient,
 		s.pubsub,
 	)
-	gqlDirectives := generated.DirectiveRoot{HasRole: func(ctx context.Context, obj interface{}, next graphql.Resolver, role model.Role) (res interface{}, err error) {
-		hasRole, err := services.HasRole(ctx, role)
-		if err != nil {
-			return nil, err
-		}
-		if !hasRole {
-			return nil, errors.New("not authorized")
-		}
-		return next(ctx)
-	}}
+	gqlDirectives := generated.DirectiveRoot{
+		HasRole: func(ctx context.Context, obj interface{}, next graphql.Resolver, role model.Role) (res interface{}, err error) {
```

Comment:
Having trouble telling what this change is - is this just formatting?

Reply:
Thanks Clay! That's right, I meant to just remove the custom directive, but it looks like I saved a space in here. There is no functional change.